sklearn/feature_selection/from_model.py

```diff
@@ -227,3 +231,7 @@ def partial_fit(self, X, y=None, **fit_params):
             self.estimator_ = clone(self.estimator)
         self.estimator_.partial_fit(X, y, **fit_params)
         return self
+
+    def _more_tags(self):
```

Comment:
@jnothman @NicolasHug I updated this to inherit allow_nan from its estimator. This also required a change to SelectorMixin in feature_selection/base.py to satisfy estimator checks on transform. Also, I don't think this change fixes the need for the hack in check_estimators_pickle because that is necessary to handle the score_func's in univariate selectors, which aren't estimators and thus don't have the allow_nan tag. One option would be to add tags to those functions indicating whether they allow nan anyways. What do you think?